Assets/Scripts/Game/Controllers/PlayerController.cs

```diff
@@ -175,6 +175,10 @@ void HandleInput()
 	}
 
 	void FixedUpdate () {
+		if (Time.timeScale == 0) {
+			return;
+		}
```

Comment:
This was not in original source code, and there doesn't seem to be anything that breaks up in void FixedUpdate(), but I placed it here anyway just for convenience.